Frameworks/UIKit/UITabBarButton.mm

```diff
@@ -127,6 +127,8 @@ - (void)drawRect:(CGRect)pos {
         size = [title sizeWithFont:font constrainedToSize:CGSizeMake(0.0f, 0.0f) lineBreakMode:UILineBreakModeClip];
 
         CGRect textRect{};
+
+        // Horizontally center text
```

Comment:
double sign off!

---
In reply to: [98794256](https://github.com/Microsoft/WinObjC/pull/1882#discussion_r98794256) [](ancestors = 98794256)